ui/apps/everest/src/components/cluster-form/resources/constants.ts

```diff
@@ -68,14 +68,16 @@ export const DEFAULT_SIZES = {
   },
 };
 
-export const SHARDING_DEFAULTS = {
-  [DbWizardFormFields.shardConfigServers]: {
-    min: '1',
-    max: '7',
-  },
-  [DbWizardFormFields.shardNr]: {
-    min: '1',
-  },
+export const DEFAULT_CONFIG_SERVERS = ['1', '3', '5', '7'];
+
+export const MIN_NUMBER_OF_SHARDS = '1';
```

Comment:
Setting default number as strings seem counter-intuitive and prone to error.
I think we should define numbers instead, and convert to string where necessary.,